### ISSUE TITLE
create-nuxt-app doesn't work using pnpx

### ISSUE THREAD

**Issue by shaneknysh** [pnpm/pnpm#1671]
create-nuxt-app returns command not found. Possibly related to #444 

### Code to reproduce the issue:
pnpx create-nuxt-app [app-name]

### Expected behavior:
Creates nuxt app and launches the nuxt app configuration cli

### Actual behavior:
Already up-to-date
Resolving: total 346, reused 346, downloaded 0, done
command not found: create-nuxt-app

### Additional information:
 - `node -v` prints:
v11.10.0

 - Windows, OS X, or Linux?:
Windows 10 both powershell and cmd

**Comment by zkochan** [comment]
Is it still reproducible?

I could not repro it on Linux, with pnpm v3.1.1-0 and `create-nuxt-app@2.6.0`

**Comment by shaneknysh** [comment]
Still fails under windows 10 with pnpm 2.25.6 in command line and powershell.

After failure in powershell displays notification for update available for pnpm 2.25.6 -> 3.1.0.

After update to 3.1.0 still fails in powershell and command line.
```
PS> pnpx create-nuxt-app test
Already up-to-date
Resolving: total 346, reused 346, downloaded 0, done
command not found: create-nuxt-app
```

**Comment by digitalfuel** [comment]
Just came across this looking for something related.
If you use pnpx a tarball with `pnpx ./create-nuxt-app-1.1.0-alpha.2a.tgz <project>` it does not work as the directory resolves to ...\AppData\Roaming\npm\pnpm-global\2\ folder.
If you put the tarball in this directory then it starts but I then get `command not found: create-npos-app`.
Though that when using from npm eg `pnpx create-nuxt-app <project>` it works.
Win10, vscode, powershell

**Comment by shaneknysh** [comment]
working on windows 10/ powershell as of pnpm version 3.4.1

**Comment by zkochan** [comment]
Maybe this fixed it: #1844
